xbmc/music/tags/TagLibVFSStream.h

```diff
@@ -63,7 +63,7 @@ namespace MUSIC_INFO
      */
 #if (TAGLIB_MAJOR_VERSION >= 2)
     void insert(const TagLib::ByteVector& data,
-                TagLib::offset_t start = 0,
+                off_t start = 0,
```

Comment:
Why? This was correct as it was.

Reply:
Yes, I found what's wrong here :) Crafting the fix :)